### PR TITLE
buy many products scenario

### PR DESCRIPTION
**Pull request by kmatlaha**
buy many products scenario

### CODE REVIEW

File: tests/buyProducts_test.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
-const { newBillingAddressButton, confirmOrderButtonOnCheckoutPage } = require("../pages/checkout");
-const orderHistory = require("../pages/orderHistory");
+
```
Comment (adudyak) [inline tests/buyProducts_test.js]: remove empty line

File: tests/buyProducts_test.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+productLinks = new DataTable (['link']);
+productLinks.add (['http://opencart.qatestlab.net/index.php?route=product/product&path=31&product_id=40']);
+productLinks.add (['http://opencart.qatestlab.net/index.php?route=product/product&path=31&product_id=43']);
+productLinks.add (['http://opencart.qatestlab.net/index.php?route=product/product&path=31&product_id=47']);
+productLinks.add (['http://opencart.qatestlab.net/index.php?route=product/product&path=31&product_id=74']);
```
Comment (adudyak) [inline tests/buyProducts_test.js]: remove DataTable

File: tests/buyProducts_test.js [hunk 3]
```diff
@@ -16,12 +20,20 @@ let billingDetails = {
     postcode: 'A1A1A1',
 };
 
+const { getArrayOfProductLinkObjects } = require('../helpers/productLinksGetter');
```
Comment (adudyak) [inline tests/buyProducts_test.js]: not used - remove

File: tests/buyProducts_test.js [hunk 4]
```diff
@@ -16,12 +20,20 @@ let billingDetails = {
...
+const LinksGetter = require('../helpers/productLinksGetter');
+let productLinks2 = LinksGetter.getLinks();
```
Comment (adudyak) [inline tests/buyProducts_test.js]: name variable `productLinks`

**Review by adudyak — CHANGES_REQUESTED** [review]

File: tests/buyProducts_test.js (hunk 1 above)
Comment (kmatlaha) [reply tests/buyProducts_test.js]: I did not find these lines in my code.

File: tests/buyProducts_test.js (hunk 2 above)
Comment (kmatlaha) [reply tests/buyProducts_test.js]: Removed.

File: tests/buyProducts_test.js (hunk 3 above)
Comment (kmatlaha) [reply tests/buyProducts_test.js]: Removed.

File: tests/buyProducts_test.js (hunk 4 above)
Comment (kmatlaha) [reply tests/buyProducts_test.js]: Changed.

**Pull request merged**